_commands/basics/less.md

```diff
@@ -3,22 +3,69 @@
 
 less
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`less` is a tool to view a text file one screen at a time. You can navigate through the file with `j` or `RETURN`, and backwards with `k`
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ less hello.txt
+# displays hello.txt
+q # to quit
 ~~~
 
-<!--more-->
+<!--less-->
 
 ### Useful Options / Examples
 
-#### Example command
+Typing `/` within `less` allows you to forward search using regex
+and `?` allows you to backward search
+pressing `n` goes to the next item
+~~~ bash
+$ less hello.txt
+........
+........
+........
+..git...
+.....git
+
+:/git
+# git will be highlighted
+n
+# the next git is highlighted
+~~~ 
+
 
-##### Break it down
+#Options:
 
-#### Example command
+| Options |                         Description                        |
+|---------|:----------------------------------------------------------:|
+| **-?**  |                       Displays help.                       |
+| **-I**  |               Makes search case insensitive.               |
+| **-M**  |                    More detailed prompt.                   |
+| **-S**  | Long lines are chopped, and can be seen by side scrolling. |
+| **-X**  |      Contents of file remain on screen after exiting.      |
 
-##### Break it down
+#Commands:
+| Command         |                              Description                             |
+|-----------------|:--------------------------------------------------------------------:|
+| **Space bar**   |                               Next Page                              |
```

Comment:
Could you include the reference (either a link or another command to run) to get this information in the tables? That way people can dig deeper if they want on their own.